src/core-utils/gnc-features.c

```diff
@@ -45,6 +45,7 @@ static gncFeature known_features[] =
     { GNC_FEATURE_NUM_FIELD_SOURCE, "User specifies source of 'num' field'; either transaction number or split action (requires at least GnuCash 2.5.0)" },
     { GNC_FEATURE_KVP_EXTRA_DATA, "Extra data for addresses, jobs or invoice entries (requires at least GnuCash 2.6.4)" },
     { GNC_FEATURE_GUID_BAYESIAN, "Use account GUID as key for Bayesian data (requires at least GnuCash 2.6.12)" },
+    { GNC_FEATURE_GUID_FLAT_BAYESIAN, "Use account GUID as key for bayesian data and store KVP flat" },
```

Comment:
I think it would be useful to add the minimum required version that supports the feature in the help string. This string is displayed to the user when a file using the corresponding feature is opened with a gnucash version that doesn't support it. Setting the version string gives the user a hint as to which gnucash version to install.